d2layouts/d2sequence/sequence_diagram.go

```diff
@@ -143,6 +195,11 @@ func (sd *sequenceDiagram) placeActors() {
 //        │
 func (sd *sequenceDiagram) addLifelineEdges() {
 	endY := sd.getMessageY(len(sd.messages))
+
+	for _, note := range sd.notes {
```

Comment:
notes were already placed, and messages too.
```
endY := math.Max(lastMessage.Route[1].Y, lastNote.TopLeft.Y+lastNote.Height) + sd.yStep
```

Reply:
can't use route[1] cuz later on there may be non-instantaneous msgs, but other than that, it's a good point